src/main/java/com/solvd/micro9/scooters/messaging/serde/CustomSerdes.java

```diff
@@ -0,0 +1,29 @@
+package com.solvd.micro9.scooters.messaging.serde;
+
+import com.solvd.micro9.scooters.domain.RentEvent;
+import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.common.serialization.Serdes;
+import org.springframework.kafka.support.serializer.JsonDeserializer;
+import org.springframework.kafka.support.serializer.JsonSerializer;
+
+import java.math.BigDecimal;
+
+public final class CustomSerdes {
+
+    private CustomSerdes() {
+    }
+
+    public static Serde<RentEvent> RentEvent() {
```

Comment:
stay away from static methods and utility classes, they couple so much your components

Reply:
@h1alexbel then how to improve this design? use beans?

Reply:
done 0f62a40

Reply:
redone 92d648b